From c49d2935748d943268f86b1cbb0217ef9c236959 Mon Sep 17 00:00:00 2001
From: israfilbd <144210295+israfilbd@users.noreply.github.com>
Date: Sat, 29 Mar 2025 04:28:48 +0600
Subject: [PATCH] Mic fix

---
 .../com/android/server/policy/PhoneWindowManager.java     | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/services/core/java/com/android/server/policy/PhoneWindowManager.java b/services/core/java/com/android/server/policy/PhoneWindowManager.java
index f76c9f28da081..5f5cb32b4840f 100644
--- a/services/core/java/com/android/server/policy/PhoneWindowManager.java
+++ b/services/core/java/com/android/server/policy/PhoneWindowManager.java
@@ -6345,14 +6345,6 @@ && isWakeKeyWhenScreenOff(keyCode)) {
                 break;
             }
 
-            case KeyEvent.KEYCODE_MUTE:
-                result &= ~ACTION_PASS_TO_USER;
-                if (down && event.getRepeatCount() == 0) {
-                    notifyKeyGestureCompleted(event,
-                            KeyGestureEvent.KEY_GESTURE_TYPE_SYSTEM_MUTE);
-                    toggleMicrophoneMuteFromKey();
-                }
-                break;
             case KeyEvent.KEYCODE_MEDIA_PLAY:
             case KeyEvent.KEYCODE_MEDIA_PAUSE:
             case KeyEvent.KEYCODE_MEDIA_PLAY_PAUSE:
